src/main/java/seedu/address/model/client/Name.java

```diff
@@ -10,13 +10,13 @@
 public class Name {
 
     public static final String MESSAGE_CONSTRAINTS =
-            "Names should only contain alphanumeric characters and spaces, and it should not be blank";
+            "Names should only contain letters, numbers, spaces, hyphens, apostrophes, and it should not be blank";
```

Comment:
wanna add "no symbols" too to match explaination on the UG?

Reply:
Nice idea, i have added it!